runtime/vm/jvminit.c

```diff
@@ -2866,6 +2906,7 @@ modifyDllLoadTable(J9JavaVM * vm, J9Pool* loadTable, J9VMInitArgs* j9vm_args)
 		JVMINIT_VERBOSE_INIT_VM_TRACE(vm, "-Xint set\n");
 	}
 	if (xnoaot) {
+		vm->extendedRuntimeFlags2 |= J9_EXTENDED_RUNTIME2_ENABLE_AOT;
```

Comment:
Is this correct ? setting J9_EXTENDED_RUNTIME2_ENABLE_AOT when xnoaot is found.

Reply:
Sorry I just fixed this. Should be under xaot.

Reply:
I'm going to double check the logic here again